### ISSUE TITLE
Seeing error 'WebpackError: Cannot find react-intl/locale-data/en'

### ISSUE THREAD

**Issue by dianakhuang** [wiziple/gatsby-plugin-intl#36]
When running `gatsby-plugin-intl@0.2.4`, `gatsby@2.10.4`, and `react-intl@2.9.0`, I'm seeing this error on both the development server and the static build.

I played around with versions and did not see this with `gatsby-plugin-intl@0.1.7`.

I am also aware of the fact that [there are still issues](https://github.com/gatsbyjs/gatsby/issues/15101) with `gatsby@2.10.4`, but this doesn't seem related.

```
  36 |   if (!localeData) {
> 37 |     throw new Error("Cannot find react-intl/locale-data/" + language);
     | ^
  38 |   }
  39 |
  40 |   _reactIntl.addLocaleData.apply(void 0, localeData);


  WebpackError: Cannot find react-intl/locale-data/en

  - wrap-page.js:37 addLocaleDataForGatsby
    node_modules/gatsby-plugin-intl/wrap-page.js:37:1

  - wrap-page.js:106 Object._default [as wrapPageElement]
    node_modules/gatsby-plugin-intl/wrap-page.js:106:1


  - gatsby-ssr.js:13 Module.replaceRenderer
```

gatsby-config.js
```
    {
      resolve: 'gatsby-plugin-intl',
      options: {
        // language JSON resource path
        path: `${__dirname}/src/i18n/messages`,
        // supported language
        languages: ['es'],
        defaultLanguage: 'en',
        // if true, redirect to `/en` when connecting `/`
        redirect: false,
      },
    },
```

Gatsby Info:
```

  System:
    OS: Linux 4.9 Debian GNU/Linux 9 (stretch) 9 (stretch)
    CPU: (6) x64 Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz
    Shell: 4.4.12 - /bin/bash
  Binaries:
    Node: 10.15.3 - /usr/local/bin/node
    Yarn: 1.13.0 - /usr/local/bin/yarn
    npm: 6.4.1 - /usr/local/bin/npm
  Languages:
    Python: 2.7.13 - /usr/bin/python
  npmPackages:
    gatsby: 2.10.4 => 2.10.4
    gatsby-cli: ^2.4.15 => 2.7.2
    gatsby-image: ^2.0.33 => 2.2.3
    gatsby-link: ^2.0.16 => 2.2.0
    gatsby-node-helpers: ^0.3.0 => 0.3.0
    gatsby-plugin-intl: 0.2.4 => 0.2.4
    gatsby-plugin-netlify: ^2.0.12 => 2.1.0
    gatsby-plugin-netlify-cms: ^3.0.14 => 3.0.18
    gatsby-plugin-purgecss: ^4.0.0 => 4.0.0
    gatsby-plugin-react-axe: ^0.2.2 => 0.2.2
    gatsby-plugin-react-helmet: ^3.0.9 => 3.1.0
    gatsby-plugin-react-svg: ^2.1.1 => 2.1.1
    gatsby-plugin-sass: ^2.0.11 => 2.1.0
    gatsby-plugin-segment-js: ^3.0.1 => 3.0.1
    gatsby-plugin-sharp: ^2.0.28 => 2.2.1
    gatsby-plugin-web-font-loader: ^1.0.4 => 1.0.4
    gatsby-plugin-webpack-bundle-analyser-v2: ^1.1.0 => 1.1.1
    gatsby-source-filesystem: ^2.0.25 => 2.1.1
    gatsby-transformer-json: ^2.1.10 => 2.2.0
    gatsby-transformer-remark: ^2.3.3 => 2.5.0
  npmGlobalPackages:
    gatsby: 2.4.2
```

**Comment by dianakhuang** [comment]
It seems like this Webpack add-on doesn't consider that `defaultLanguage` might not be in the list of languages: https://github.com/wiziple/gatsby-plugin-intl/blob/master/src/gatsby-node.js#L5

We would like to not have separate `en` paths for our site, which is why `en` isn't included in the languages list.

**Comment by wiziple** [comment]
published on `gatsby-plugin-intl@0.2.5`
